### ISSUE TITLE
Center shows as NA in table but renders correctly on graph

### ISSUE THREAD

**Issue by mwgravitt** [tomhopper/qcc_ggplot#8]
I am seeing "NA" as the Center value even though the Center line renders correctly on the graph.

![image](https://cloud.githubusercontent.com/assets/8877170/17931485/be63b430-69d9-11e6-873c-2f11a1437b63.png)

**Comment by mwgravitt** [comment]
I have more information on how to reproduce this.  It only occurs when I pass "label.limits=" to qcc.  Here's the command that I run that has the issue:

 x<-qcc (retailDataSet$Time.to.Delivery, 
       type="xbar.one", 
       labels=retailDataSet$WE.Date,
       nsigmas=2,
       label.limits=c('2S', '2S'),
       title=paste(retailerName, "Cycle Time", sep=" "),
       xlab = ('Week Ending'),
       ylab=('Minutes')
    )

with this result:

![image](https://cloud.githubusercontent.com/assets/8877170/18057182/e83fdd18-6ddc-11e6-8009-2ccf11df4816.png)

Note that the center is "NA" and the LCL and UCL use the labels that I passed in.  In base qcc, the labels are used on the horizontal lines but not in the grid below.

If I comment out the line that passes "label.limits", it works:

![image](https://cloud.githubusercontent.com/assets/8877170/18057281/52a33d8a-6ddd-11e6-82a9-5d22b6e2af69.png)

As a workaround, I changed lines 461, 490, and and 508 to the below, respectively:
      grid.text(round(center),
      grid.text(round(limits[1]),
      grid.text(round(limits[2]),

thanks
